taxonomies/ingredients.txt

```diff
@@ -6735,6 +6753,27 @@ fr:fond de sauce
 
 fr:érythrosine
 
+en:vegetal oil
+fr:huile végétale
```

Comment:
i think there's another en:vegetal oil, vegetal oils
fr:huile végétale, huiles végétales that i added

Reply:
not sure